logstash-core/logstash-core.gemspec

```diff
@@ -14,12 +14,11 @@ Gem::Specification.new do |gem|
   gem.files         = Dir.glob(["logstash-core.gemspec", "lib/**/*.rb", "spec/**/*.rb", "locales/*", "lib/logstash/api/init.ru"])
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "logstash-core"
-  gem.require_paths = ["lib"]
+  gem.require_paths = ["lib", "vendor/jars"]
```

Comment:
the reason it is in `vendor/jars` is specifically since `require_jar` from the `jar-dependencies` library looks in that directory. Once we write our own, I will change this. I think I'll leave it as is for now